libs/ledger-live-common/src/featureFlags/types.ts

```diff
@@ -6,7 +6,7 @@ export type FeatureId =
   | "ratings"
   | "counterValue"
   | "buyDeviceFromLive"
-  | string;
```

Comment:
leaving string was not strict enough in my opinion, because of that for instance we didn't detect that a default config for llmUsbFirmwareUpdate was missing